data/iedb_jespersen_et_al/meta.yaml

```diff
@@ -0,0 +1,67 @@
+name: iedb_jespersen_et_al
+description: |-
+  Epitope prediction is to predict the active region in the antigen.
+  This dataset is from Bepipred, which curates a dataset from IEDB. It collects B-cell
+  epitopes and non-epitope amino acids determined from crystal structures.
+targets:
+- id: active_position
+  description: amino acids sequence position that is active in binding
+  units: ''
+  type: categorical
+  names:
+  - amino acids sequence active in binding
```

Comment:
so the target is a sequence and the input is the full sequence?

Reply:
> so the target is a sequence and the input is the full sequence?
Yeah, I forget to comment on that. Input was sequence and target is sequence as well we could make it categorical, but with a lot of category. I don't know what is your opinion of that.

Reply:
Yes, this is how it is currently implemented. I set it up to only give the corresponding epitope sequence without the `_` for the other parts of the sequence.